src/components/images_uploading/DisplayImageList.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { fetchImages } from "./../../actions";
+
+class DisplayImageList extends Component {
+  componentDidMount() {
+    this.props.fetchImages();
+  }
+  render() {
+    const { images } = this.props;
+
+    return (
+      <ol>
+        {images.map(image => {
+          return (
+            <>
+              {/* Warning: Warning: Each child in a list should have a unique "key" prop.

```

Comment:
I've added 'key' here, but there's still this warning in my browser console.... If you know why, pls let me know :)

Reply:
Make sure that s3key is not null or try to use image._id instead.